apple-nav/src/components/SubItem.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+const SubItem = props => {
+  const subItemName = props.match.params.subItem.split("-").join(" ");
+  const currentLocation = props.location.pathname.split("/")[1];
```

Comment:
Good job using your react-router props (match and location) to make sure the image that is displayed is the correct one.